provider/cmd/status.go

```diff
@@ -50,9 +53,21 @@ func doStatus(cmd *cobra.Command) error {
 		return err
 	}
 
-	if err = cctx.PrintOutputLegacy(result); err != nil {
+	marshaled, err := json.Marshal(result)
+	if err != nil {
+		return err
+	}
+
+	buf := &bytes.Buffer{}
+	err = json.Indent(buf, marshaled, "", "  ")
+	if err != nil {
+		return err
+	}
+	_, err = buf.WriteRune('\n')
+	if err != nil {
 		return err
 	}
 
-	return nil
+	_, err = fmt.Print(buf.String())
```

Comment:
you can use `os.Stdout.Write(buf)` to avoid converting to a string.

Reply:
Good point, change made